ui/widgets/nuxeo-uploader-behavior.js

```diff
@@ -419,6 +419,12 @@ export const UploaderBehavior = {
 
   _batchFailed(error) {
     this.uploading = false;
+    this.files.forEach((file, index) =>
+      this._updateFile(index, {
+        error: file.progress < 100 ? error : undefined,
```

Comment:
shouldn't we check for `file.complete` instead of `file.progress < 100`?

Reply:
I guess we could!

Reply:
Also thought about that, but since it seems that we [set them at the same time ](https://github.com/nuxeo/nuxeo-elements/blob/0055ef0e3e55833951ce62d8c20d60396d5eb351/ui/widgets/nuxeo-uploader-behavior.js#L439-L445), both should be ok 🙂